### PR TITLE
[master]zcc supports Sles15 as guest

### PR DESCRIPTION
**Pull request by wanxiangyuer**
add schema validation for sles15
implement the sles15 distro class

### CODE REVIEW

**Comment by wanxiangyuer** [comment]
The result of `tox -v`:
```
copying static files... ... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded.

The HTML pages are in doc/build/html.
__________________________________________________________________________ summary __________________________________________________________________________
  pep8: commands succeeded
  py27: commands succeeded
  py36: commands succeeded
  docs: commands succeeded
  congratulations :)
```

File: zvmsdk/sdkwsgi/validation/parameter_types.py [hunk 1]
```diff
@@ -381,7 +381,7 @@ def valid_char(char):
  '^((r|R)(e|E)(d|D)(h|H)(a|A)(t|T))(6|7){1}([.][0-9])?$'},
 {'type': 'string',
  'pattern':
- '^((s|S)(l|L)(e|E)(s|E))(11|12){1}(([.]|((s|S)(p|P)))[0-9])?$'},
+ '^((s|S)(l|L)(e|E)(s|E))(11|12|15){1}(([.]|((s|S)(p|P)))[0-9])?$'},
 {'type': 'string',
  'pattern':
  '^((s|S)(u|U)(s|S)(e|E))(11|12){1}(([.]|((s|S)(p|P)))[0-9])?$'},
```
Comment (dongyanyang) [inline zvmsdk/sdkwsgi/validation/parameter_types.py:387]: pls also update this suse pattern, to keep suse and sles consistent.

File: zvmsdk/dist.py [hunk 2]
```diff
@@ -1497,6 +1497,11 @@ def _enable_network_interface(self, device, ip, broadcast):
         return activeIP_str
 
 
+class sles15(sles12):
```
Comment (jichenjc) [inline zvmsdk/dist.py:1500]: nothing special for SLES15 , correct? e.g files or systemd units?

Comment (wgxoyun) [reply zvmsdk/dist.py:1500]: So far, I have tested the deploy/start/stop/delete vm, the sles15 class inherited from sles12 can work well. Subsequently, maybe some special method will be added to sles15 class to support more operations such as attach/detach volume. etc.

**Comment by jichenjc** [comment]
please rebase

**Comment by jichenjc** [comment]
@wanxiangyuer please rebase...

**Pull request merged**